gradio/strings.py

```diff
@@ -10,7 +10,7 @@
     "RUNNING_LOCALLY": "Running on local URL:  {}",
     "RUNNING_LOCALLY_SEPARATED": "Running on local URL:  {}://{}:{}",
     "SHARE_LINK_DISPLAY": "Running on public URL: {}",
-    "COULD_NOT_GET_SHARE_LINK": "\nCould not create share link, please check your internet connection.",
+    "COULD_NOT_GET_SHARE_LINK": "\nCould not create share link. Please check your internet connection or our status page: https://status.gradio.app",
```

Comment:
I wonder if we should link to this somewhere on the website, too?

Reply:
Good point, we could add a link in the footer next to the logo: 

<img width="462" alt="image" src="https://user-images.githubusercontent.com/1778297/221622754-8282ba96-455d-4683-92d1-38355b34bb14.png">

Reply:
![image](https://user-images.githubusercontent.com/1778297/221622953-d0f5f8ad-05a9-4e79-92b4-bda5a7e617ff.png)

Reply:
Added it at the bottom 

<img width="1296" alt="Screen Shot 2023-02-27 at 9 24 23 AM" src="https://user-images.githubusercontent.com/9021060/221636414-3d8f3b56-d70b-49b3-b5d6-09c3e7a11ac3.png">

Reply:
Hmm not sure but maybe it should be separated a bit more? #notagraphicdesigner but
![image](https://user-images.githubusercontent.com/1778297/221640123-203e06bf-0f57-4f7f-a3f1-eae16c51ce40.png)

Reply:
Or maybe even having it aligned on the right next to twitter / github makes would look nicer color-theme-wise

Reply:
LGTM!